lib/requester/request-wrapper.js

```diff
@@ -78,16 +79,18 @@ var _ = require('lodash'),
     setProxy = function(options, cb) {
         var url = _.get(options, 'request.url');
 
-        if (url && options.proxyManager && options.proxyManager.getProxyConfiguration) {
+        if (url && options.proxyList) {
+            if(!_.isFunction(options.proxyList.isProxyConfigList)) {
+              options.proxyList = new ProxyConfigList({}, options.proxyList);
+            }
             _.isFunction(url.toString) && (url = url.toString());
-            return options.proxyManager.getProxyConfiguration(url, function(err, obj) {
-                options.proxy = obj && obj.proxy;
-                options.tunnel = obj && obj.tunnel;
-                _.set(options, 'request._._postman_proxy', obj);
```

Comment:
@czardoz 
`_.set(options, 'request._._postman_proxy', obj);` I removed this by mistake, may i know why this is for?